src/api/QGCCorePlugin.cc

```diff
@@ -445,6 +445,6 @@ QString QGCCorePlugin::stableVersionCheckFileUrl() const
     // Custom builds must override to turn on and provide their own location
     return QString();
 #else
-    return QString("https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGC.version.txt");
+    return QString();//"https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGC.version.txt");
```

Comment:
is that on purpose ?

Reply:
Yes.  When this method returns an empty string, QGC won't check for a new version.  Not super clean, but it does the trick.

Side note: in the future we need to move to the `CUSTOM_BUILD` setup.